addons/account/views/account_move_views.xml

```diff
@@ -933,8 +931,9 @@
                                                     and [('sale_ok', '=', True), '|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]
                                                     or [('purchase_ok', '=', True), '|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]
                                                "/>
-                                        <field name="name" widget="section_and_note_text" attrs="{'required': [('parent.move_type', 'not in', ('in_invoice', 'in_refund','in_receipt'))]}"/>
```

Comment:
Why isn't it required anymore in those cases? Note sure I see the link with the purpose of the commit

Reply:
Because of the specs. The fiduciaries do not want to encode this.
Update: just discussed with @chklop, we do not want this field to be required in any case. This freedom will be useful for the OCR too.

Reply:
While it's true that you shouldn't send Customer Invoices without labels on your invoice lines, there are cases where those Invoices are managed outside of Odoo and are just imported in Odoo for Accounting purposes. Fiduciaries are the obvious cases, but there must be other clients not using Odoo for their Invoicing, only for Accounting.
In those cases, they do not care about the line labels. They might even use the OCR to scan those "external" invoices, and in that situation, in most cases the OCR will just create one line with a dumb label.
For all those cases, not having a label are valid use cases for invoices encoding and we should stop blocking the user if he doesn't use a label (or invent/create dumb labels automatically as we do in the OCR).